### ISSUE TITLE
Suspension of the AdMob SDK use for Windows Phone 8

### ISSUE THREAD

**Issue by calebeaires** [floatinghotpot/cordova-admob-pro#443]
I have received an Admob email about it today. Is there some real cordova alternative to Windows Phone 8 and WP 10 devices?

**Comment by floatinghotpot** [comment]
Could you paste the email content here please? I wanna know more about the facts.

**Comment by calebeaires** [comment]
Its content is on Portuguese.


Prezado editor,
Enviamos este e-mail porque você tem um aplicativo que usa o SDK da AdMob para Windows Phone 8. No dia 1 de novembro, suspenderemos o uso do SDK da AdMob para Windows Phone 8.

**O que acontecerá após a suspensão do uso do SDK da AdMob para Windows Phone 8?**

- Você não poderá mais criar blocos de anúncios para o Windows Phone na interface da AdMob

- Você não poderá mais fazer o download do SDK da AdMob para Windows Phone 8

- Serão removidos os exemplos e a documentação para desenvolvedores da AdMob para Windows Phone 8, bem como a categoria do Windows Phone 8 no fórum de desenvolvedores do SDK dos anúncios para dispositivos móveis do Google.

**O que acontecerá com meus aplicativos que já estão em produção?**

No momento, não há planos de bloqueio da veiculação de anúncios em aplicativos para Windows Phone 8 existentes até o dia 1 de novembro. No entanto, não podemos garantir a veiculação de anúncios após essa data.

Se você tiver alguma dúvida, entre em contato no fórum de desenvolvedores.

Fique atento a mais atualizações e dicas.

Atenciosamente,
Equipe da AdMob

**Comment by floatinghotpot** [comment]
Yeah, I see. We cannot create ad unit id for wp8 now. Most Ad networks provide iOS/Android SDK, and Google is the only one providing wp8 SDK, but not update the SDK for nearly 2 years. Now they decide no longer support it.

I will check whether Microsoft provides any Ad SDK or monetization solution, if yes, I will try writing one.

**Comment by calebeaires** [comment]
Since Windows Phone 10 has arrived, maybe you could focus on it. I have looking for one alternative since Admob will no more be there. I found out that [Smaato](https://wiki.smaato.com/display/SPX/Windows+Phone) is on good option

**Comment by calebeaires** [comment]
Is there some good news about this issue?
